MovieApp/PresentationLayer/Components/BaseSearchField/BaseSearchField.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+class BaseSearchField: UITextField {
+
+    static let defaultHeight = 40
+
+    override func leftViewRect(forBounds bounds: CGRect) -> CGRect {
+        CGRect(x: 0, y: 0, width: BaseSearchField.defaultHeight, height: BaseSearchField.defaultHeight)
```

Comment:
You can define `static let defaultSize = CGSize(width: 40, height: 40)` and then have here `CGRect(origin: .zero, size: defaultSize)` and `CGRect(origin: CGPoint(Int(bounds.maxX) - dimension, y: 0), size: defaultSize)` below.